geojson_pydantic/features.py

```diff
@@ -62,3 +73,16 @@ def __getitem__(self, index):
     def __geo_interface__(self):
         """GeoJSON-like protocol for geo-spatial (GIS) vector data."""
         return self.dict()
+
+    @classmethod
+    def validate(cls, value):
+        """Validate input."""
+        try:
+            value = json.loads(value)
+        except TypeError:
+            try:
+                return cls(**value.dict())
+            except (AttributeError, ValidationError):
+                pass
```

Comment:
I don't think that a FeatureCollection object will ever be validated. 
If it does, I don't know which path can lead to this behaviour, and a test for this is missing.

```suggestion
            pass
```

Reply:
🤷‍♂️ I don't really understand, I `copy/pasted` this to follow the same logic as what you did for GeometryCollection

Reply:
Ok I see. Then the issue is on `Feature`. I'll comment it.